modules/common/src/main/scala/notebook/front/widgets/package.scala

```diff
@@ -1,5 +1,7 @@
 package notebook.front
 
+import org.apache.spark.sql.DataFrame
```

Comment:
Sadly, won't work with _pre-df_ like spark 1.2.
But since, the code below is not necessary anymore, we can also get rid of this too